spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/pom.xml

```diff
@@ -24,6 +24,10 @@
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-gcp-starter-config</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-context</artifactId>
+        </dependency>
```

Comment:
If you need to add that manually in the sample, something is broken in a starter IMO

Reply:
`@RefreshScope` requires this dependency. LMK if I can use a more minimal one.